### ISSUE TITLE
🐛 Bug Report: The icon of the `include Auth module` doesnt load on service creation wizard

### ISSUE THREAD

**Issue by levivannoort** [amplication/amplication#7047]
### What happened?

The icon for the `include Auth module` setting on the `auth setting` page of the service creation wizard takes too long to load - if it even does.

![image](https://github.com/amplication/hacktoberfest-issues/assets/73097785/17fd6943-cd85-4a71-8e62-544c9aa71be3)

### What you expected to happen

The icon to load in as quick as the other.

### How to reproduce

Create a new service

### Amplication version

1.8.11

**Comment by ptdatta** [comment]
@levivannoort  @souravjain540 I want to work on it. Please assign it to me.

**Comment by souravjain540** [comment]
@ptdatta assigned

**Comment by souravjain540** [comment]
@ptdatta updates?

**Comment by ptdatta** [comment]
Yeah I'm working on it. Will submit a PR today. Delay due to collage😅😅
